neoforge/src/main/resources/META-INF/neoforge.mods.toml

```diff
@@ -0,0 +1,29 @@
+modLoader = "javafml"
+loaderVersion = "${loader_version_range}"
+issueTrackerURL = "https://github.com/Wynntils/Artemis/issues"
+license = "LGPL-3.0-only"
+
+[[mods]]
+modId = "wynntils"
+version = "${mod_version}"
+displayName = "Wynntils"
+authors = "Team Wynntils"
+description = "The best Wynncraft mod you'll probably find!"
+displayURL = "wynntils.com"
+logoFile = "assets/wynntils/logo.png"
+
+[[dependencies.wynntils]]
+modId = "minecraft"
+mandatory = true
+versionRange = "[${minecraft_version}]"
+
+[[dependencies.wynntils]]
+modId = "neoforge"
+mandatory = true
+versionRange = "${neoforge_version_range}"
+
+[[mixins]]
+config = "wynntils.mixins.json"
+
+[[mixins]]
+config = "wynntils.mixins.neoforge.json"
```

Comment:
Sorry, late review. This file is missing a trailing newline.
(Why don't we check for that? Maybe we ignore toml files in Spotless?)